tests/e2e/specs/shopper/cart-checkout-translations.test.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Internal dependencies
+ */
+import { cli } from '../../../utils/run-cli-from-test';
+import { merchant } from '../../../utils/merchant';
+import { shopper } from '../../../utils/shopper';
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 ) {
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( 'Skipping Checkout tests', () => {} );
+}
+
+describe( 'Shopper → Cart → Can view translated cart & checkout blocks', () => {
+	// We need to install the language files for the blocks plugin.
+	// We also need to install the plugin from w.org via the cli. This is because
+	// on w.org, the slug is `woo-gutenberg-products-block` where as here it's
+	// `woocommerce-gutenberg-products-block`. If we try to install the language files
+	// directly, it won't find them because of the slug mismatch.
+	beforeAll( async () => {
+		const resultInstallBlocks = await cli(
+			'npm run wp-env run tests-cli "wp plugin install woo-gutenberg-products-block"'
+		);
+		const resultInstallLanguages = await cli(
+			'npm run wp-env run tests-cli "wp language plugin install woo-gutenberg-products-block fr_FR"'
+		);
+		expect( resultInstallBlocks.code ).toEqual( 0 );
+		expect( resultInstallLanguages.code ).toEqual( 0 );
+		await merchant.changeLanguage( 'fr_FR' );
+	} );
+
+	// We need to clean up here by changing the language back to English
+	// and uninstalling the w.org version of Woo Blocks plugin and the language files
+	afterAll( async () => {
+		await merchant.changeLanguage( 'en_EN' );
+		const resultUninstallBlocks = await cli(
+			'npm run wp-env run tests-cli "wp plugin uninstall woo-gutenberg-products-block"'
+		);
+		const resultUninstallLanguages = await cli(
+			'npm run wp-env run tests-cli "wp language plugin uninstall woo-gutenberg-products-block fr_FR"'
+		);
+		expect( resultUninstallBlocks.code ).toEqual( 0 );
+		expect( resultUninstallLanguages.code ).toEqual( 0 );
+	} );
+
+	it( 'should be able to view translated Cart block ', async () => {
+		// [x] Install woo-blocks plugin using CLI
+		// [x] Install language for all plugins
+		// [x] Navigate to the Settings page
+		// [x] Change language to German
+		// [x] Navigate to Shop page
+		// [x] Add product to cart
+		// [x] Go to CArt page and check all translated fields
+		// [] Go toCheckout and check all translated fields
```

Comment:
Guess we can remove these comments, can't we? I also noticed that the comment says `Change language to German`, but the test itself is using French.

Reply:
Oh yeah, good spot, thanks!